hack/examples/bases/aws/kustomization.yaml.tmpl

```
@@ -32,6 +32,15 @@ patches:
         - name: "clusterConfig"
           value:
             aws: {}
+    - op: "add"
+      path: "/spec/topology/variables/0/value/addons"
+      value:
+        cni:
+          provider: calico
+        nfd: {}
+        csi:
+          providers:
+            - name: aws-ebs-sc
```

If "sc" is for StorageClass lets drop it. Its deploying more than just the StorageClass.
```suggestion
            - name: aws-ebs
```